tests/eko/scale_variations/test_expanded.py

```diff
@@ -64,20 +63,50 @@ def scheme_diff(g, k, pto, is_singlet):
         the accuracy of singlet quantities is slightly worse.
         """
         if pto[0] >= 2:
```

Comment:
do we really want a `>=` here? the difference strictly depends on the order - also I don't believe we want the `diff +=`, but instead `diff =`

Reply:
As the test is build, this is correct with `+=`.